src/os/unix.rs

```diff
@@ -9,6 +9,13 @@ use std::convert::TryInto;
     target_os = "illumos",
 ))]
 use std::env;
+#[cfg(target_os = "macos")]
+use std::os::{
+    raw::{c_long, c_uchar},
+    unix::ffi::OsStrExt,
+};
+#[cfg(not(target_os = "illumos"))]
```

Comment:
The `#[cfg]` attribute should match the block where it's used.

```suggestion
#[cfg(any(
    target_os = "linux",
    target_os = "macos",
    target_os = "dragonfly",
    target_os = "freebsd",
    target_os = "netbsd",
    target_os = "openbsd",
))]
```